src/common/TokenRescuer.sol

```diff
@@ -0,0 +1,94 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.13;
+
+import {IERC20} from "forge-std/interfaces/IERC20.sol";
+import {IERC721} from "forge-std/interfaces/IERC721.sol";
+import {IERC1155} from "forge-std/interfaces/IERC1155.sol";
+
+abstract contract TokenRescuer {
```

Comment:
Do we need a native currency withdraw here too?

Reply:
Yeah that's a great question.

I think we should. LB already has an emergency logic for withdrawal and I should rewire to use the internal rescue function on this contract. 

Good change